README.md

```diff
@@ -1,13 +1,13 @@
 # Contribute[![Build Status](https://api.travis-ci.org/NIT-dgp/contribute.svg?branch=master)]
 
-Have you been looking for open source repositories to contribute? Look no further! You have come to right place. This gem will find active repositories from github to eat away your weekend time. Say goodbay to fruitless late night research, tiring click surfing and the voice in your head which make you feel guilty for not doing enough. Be warned! Your life will never be same after you have installed this gem. We offer all these unbelievbale features at a never before price of FREE! FREE! FREE!
-Why are you still waiting? Get your terminals rolling, install this gem and experience the change.
+Have you been looking for open source repositories to contribute :smile:?Look no further :wink:! You have come to right place. This :small_blue_diamond: gem will find active repositories from github to eat away your weekend time:clock2:. Say goodbye :smiley: to fruitless late night research :weary:, tiring click surfing and the voice in your head which make you feel guilty for not doing enough. Be warned :exclamation:Your life will never be same after you have installed this gem. We offer all these unbelievbale features at a never before price of:tada: :sparkles: FREE! FREE! FREE! :sparkles:
```

Comment:
Ok @ananyo2012 The Readme was written in such a way that it felt like it could use them all.
But i guess you are correct, so i'll do as you say.

Also i made another commit and PR on https://github.com/NIT-dgp/contribute/issues/20
before i saw this review so i'll make the changes in the next commit

Thanks for reviewing :smile: